### PR TITLE
CI: fix docker build job name

### PR DESCRIPTION
**Pull request by maxknv**
#do_not_test

### Changelog category (leave one):
- CI Fix or Improvement (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
- arm and amd docker build jobs use similar job names and thus overwrite job reports
- aarch64 and amd64 suffixes added to fix this

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 25d219d91b7af1441c3bd424a1b7b4e21def0c51 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60554/25d219d91b7af1441c3bd424a1b7b4e21def0c51/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: tests/ci/docker_images_check.py [hunk 1]
```diff
@@ -173,6 +173,11 @@ def parse_args() -> argparse.Namespace:
 
 
 def main():
+    NAME = "Push to Dockerhub " + (
+        "aarch64"
+        if any(arch in platform.processor() for arch in ("aarch", "arm"))
+        else "amd64"
+    )
```
Comment (Felixoid) [inline tests/ci/docker_images_check.py:180]: we can simplify it
```suggestion
    NAME = f"Push to Dockerhub {platform.machine()}"
```

Much shorter, and good enough

Comment (maxknv) [reply tests/ci/docker_images_check.py:180]: we have aarch64 and amd64 in all other similar cases

Comment (Felixoid) [reply tests/ci/docker_images_check.py]: No problem, let's use `args.suffix` without any other logic

Comment (Felixoid) [reply tests/ci/docker_images_check.py]: `parser.add_argument("--suffix", type=str, required=True, help="arch suffix")` is already there

Comment (maxknv) [reply tests/ci/docker_images_check.py]: what is the problem with current implementation? I don't think we need input parameter to define job name

Comment (maxknv) [reply tests/ci/docker_images_check.py]: > we have aarch64 and amd64 in all other similar cases
with this I meant that processor.machine() gives suffixes different from what we use in ci. that's why there is that additional if/else to map it into aarch64/amd64

Comment (Felixoid) [reply tests/ci/docker_images_check.py]: The problem it doesn't use the info we have and adds the unnecessary differences between https://github.com/ClickHouse/ClickHouse/blob/master/.github/workflows/reusable_docker.yml#L42 + https://github.com/ClickHouse/ClickHouse/blob/master/.github/workflows/reusable_docker.yml#L27

We have it already. Let's use it

**Review by Felixoid — APPROVED** [review]

**Pull request merged**